assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -29,6 +30,7 @@ import { getSubsLinks } from '../helpers/externalLinks';
 
 import type { SubsUrls } from '../helpers/externalLinks';
 import type { Participations } from '../../../helpers/abtest';
+import type { ReferrerAcquisitionData } from '../../../helpers/tracking/acquisitions';
```

Comment:
I would change this import to be: 
`import type { ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';`

In that way, it is clearer that is a _shared_ helper. Additionally, would you be able to update the `abtest` import?

Reply:
Done. In dotcom I used `declare type` (in the `flow-typed` directory) for these kind of definitions, so they're available throughout the project - a similar thing could be useful here. 

https://github.com/guardian/frontend/blob/master/static/src/javascripts/__flow__/types/ab-tests.js